Xbehave.sln

```diff
@@ -43,36 +37,43 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		build.csx = build.csx
 		CONTRIBUTING.md = CONTRIBUTING.md
 		Dictionary.xml = Dictionary.xml
+		global.json = global.json
 		LICENSE = LICENSE
 		README.md = README.md
 		stylecop.json = stylecop.json
 	EndProjectSection
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Xbehave.Core", "src\Xbehave.Core\Xbehave.Core.xproj", "{37910B4A-996D-4DB1-816E-7ABAAE43CEC2}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Xbehave.Execution", "src\Xbehave.Execution\Xbehave.Execution.xproj", "{27AB17E5-1796-4DBD-8FFE-99249114C93A}"

```

Comment:
I never in a million years would've figured out the new name. Even looking at it, I'm not sure why.